2765. Longest Alternating Subarray/solution.py

```diff
@@ -11,39 +10,81 @@ def alternatingSubarray(self, nums: List[int]) -> int:
 
     	# declare a subArraySize variable that keeps track of length of 
     	# alternating subarrays.
+        subArraySize = 0
 
-    	# declare a list to hold potential subArraySizes.
-
+    	# variable to hold the largest size
```

Comment:
In our language, we often don't use "variable" for comments. A comment isn't actually necessary here. Maybe "keeps track of the largest subarray size" is appropriate, but "largest size" is implied by the variable name.